dct_backend/dct_capacity.py

```diff
@@ -0,0 +1,14 @@
+class Capacity():
```

Comment:
I'm not entirely convinced as to why this class might come in handy -- is there a reason you went with it that I might be overlooking?

Reply:
I think this was one of the first things I wrote so I just kept it throughout, it does seem like I really don't need it lol